addons/web/static/src/js/widgets/dropdown_menu.js

```diff
@@ -15,6 +15,11 @@ var DropdownMenu = Widget.extend({
         'click .o_item_option': '_onOptionClick',
         'click span.o_trash_button': '_onTrashClick',
         'hidden.bs.dropdown': '_onBootstrapClose',
+        'click div.dropdown-item-text': function (ev) {
+            // Protect against Bootstrap dropdown close from inside click
+            ev.preventDefault();
+            ev.stopPropagation();
+        }
```

Comment:
- `.dropdown-item-text` can be other things than `div`
- Only `preventDefault` or `stopPropagation` is needed here (probably the second one)
- Not using an inline function would be better
- I have no problem reviewing this but the framework team should handle those kind of bug review normally ;)